tests/e2e_test.go

```diff
@@ -91,24 +116,14 @@ var _ = ginkgo.BeforeSuite(func() {
 		fundedKey,
 	)
 
-	var ctx context.Context
-	ctx, cancelDecider = context.WithCancel(context.Background())
-	// we'll call cancelDecider in AfterSuite, but also call it if this
-	// process is killed, because AfterSuite won't always run then:
-	signalChan := make(chan os.Signal, 2)
-	signal.Notify(signalChan, os.Interrupt, syscall.SIGTERM)
-	go func() {
-		<-signalChan
-		cancelDecider()
-	}()
 	decider = exec.CommandContext(ctx, "./tests/cmd/decider/decider")
 	decider.Start()
-	go func() { // panic if the decider exits abnormally
+	go func() {
 		err := decider.Wait()
-		// Context cancellation is the only expected way for the
-		// process to exit, otherwise panic
+		// Context cancellation is the only expected way for the process to exit
+		// otherwise log an error but don't panic to allow for easier cleanup
 		if !errors.Is(ctx.Err(), context.Canceled) {
-			panic(fmt.Errorf("decider exited abnormally: %w", err))
+			log.Error("Decider exited abnormally: ", "error", err)
```

Comment:
Can we make these `log.Fatal`? With the avalanchego logger, fatal doesn't exit the program.

Reply:
Actually this file is using the geth logger apparently which is not instantiated but just called directly.